docs/blog/2018-06-15-why-we-built-the-site-showcase/index.md

```diff
@@ -0,0 +1,111 @@
+
+## Source code
+
+Many people learn how to build things by looking at src code for other sites. As you browse through the site showcase, look for the GitHub icon on each site’s detail page which links you to the source code (not all sites will have it):
+
+![Site Showcase Source Code Available Icon](site-showcase-source-code.png)
+
+## Showing off what Gatsby can do
+
+The site showcase can help you pitch Gatsby to clients and coworkers as the best option for their next project. As they search the site showcase, it will provide a vision of what developers can accomplish with Gatsby.
+
+# UX research process
+
+Here’s where we discussed and built the site showcase if you'd like to go through the discussion that led to this project:
+* [Site Showcase Issue](https://github.com/gatsbyjs/gatsby/issues/4392)
+* [Site Showcase WIP PR](https://github.com/gatsbyjs/gatsby/pull/5524)
+
+## Empathy map
+
+Here’s a summary of what I learned from over 80 interviews with Gatsby users. This "empathy map" describes what kinds of pain points we're trying to solve with the site showcase.
+
+![Empathy Map Site Showcase](empathy-map-site-showcase.jpg)
+ 
+## User Flow
+
+The empathy map reflects how it took most people a lot of time browsing through the Awesome Gatsby list to find example sites and how many people share their sites through Twitter and there isn’t enough time or room for everyone to get retweeted by the @Gatsbyjs account.
+
+With the empathy map and the interviews as our guide, we created a list of jobs to be done and imagined the typical flow a user would take through the site showcase. We did this to make sure we planned both a sensible information architecture and the necessary functionality to meet user’s main needs.
+
+![User flow](user-flow-site-showcase.jpg)
+ 
+## Defining the MVP
+
+Here are the features that clearly needed to be included in the MVP; you'll notice that we checked all the boxes thanks to the hard work of many people!
+
+Users should be able to:
+- [x] See screenshots of sites built with Gatsby (Landing page)
+- [x] See source code (Site detail page)
+- [x] Share sites by copying the URL
+- [x] Filter by category
+- [x] Search
+- [x] Submit their own site to the showcase through yaml file (Shannon writes the instructions)
+- [x] See featured sites
+ 
+# Ways you can help
+
+## Next steps for the site showcase
+
+* If you created a site with Gatsby or know someone who did and would like it to appear in the site showcase, [follow these instructions to add it](https://github.com/gatsbyjs/gatsby/blob/v2/docs/docs/site-showcase-submissions.md).
+* If you see a site with no description and you know who built it, kindly remind them to update it!
+* If you’d like to help speed up the previous two suggestions by writing a script to contact people who have submitted their sites in the past, see [issue #5928](https://github.com/gatsbyjs/gatsby/issues/5928): Site showcase: add descriptions, creators, and dates added
```

Comment:
* If you’d like to help us automate the process of contacting site creators, we're [looking for help with issue #5928] to do just that.